docs/examples/table/summary.vue

```diff
@@ -24,6 +24,7 @@
 </template>
 
 <script lang="ts" setup>
+import { type VNode, h } from 'vue'
```

Comment:
I see that in the code, we usually write type imports and value imports separately. Maybe it would be better to write them separately here to maintain a unified style.

Reply:
> I see that in the code, we usually write type imports and value imports separately. Maybe it would be better to write them separately here to maintain a unified style.

DONE!